### PR TITLE
adds tipHeight function

### PR DESCRIPTION
**Pull request by deweller**
This is something that I found useful

### CODE REVIEW

File: blockchain.js [hunk 1]
```diff
@@ -203,6 +203,12 @@ Blockchain.prototype.tip = function (callback) {
   })
 }
 
+Blockchain.prototype.tipHeight = function (callback) {
```
Comment (dcousens) [inline blockchain.js:206]: https://github.com/bitcoinjs/indexd/blob/master/blockchain.js#L200 ?

File: adapter.js [hunk 2]
```diff
@@ -57,6 +57,11 @@ Adapter.prototype.tip = function (callback) {
   this.blockchain.tip(callback)
 }
 
+// returns blockchain chain tip height
+Adapter.prototype.tipHeight = function (callback) {
+  this.blockchain.tipHeight(callback)
```
Comment (dcousens) [inline adapter.js:62]: Could you simply use `this.blockchain.tip((err, res) => callback(err, res && res.height))`

**Review by dcousens — APPROVED** [review]
ACK,  but wonder if we could get away with only using `.tip`?

**Comment by dcousens** [comment]
Hmmm,  wait `adapter` already exposes `tip`... *shrug*

**Comment by deweller** [comment]
`adapter` and `blockchain` already expose a `tip` function.  But this function returns only the block hash and not the entire tip object.

Changing this to return the entire tip object would be ideal, but it would break some other dependencies on the `tip` function so it would be a more complicated task.

I think I would prefer `tip` to return the entire tip object (`height` and `blockId`) - but that would be a breaking change and would require more work.

**Comment by dcousens** [comment]
> I think I would prefer tip to return the entire tip object (height and blockId) - but that would be a breaking change and would require more work.

Agreed.

ACK as is for now :+1:

**Pull request merged**